modules/gapi/src/backends/ocl/goclcore.cpp

```diff
@@ -6,10 +6,30 @@
 
 
 #include "precomp.hpp"
+#include "logger.hpp"
 
 #include <opencv2/gapi/core.hpp>
 #include <opencv2/gapi/ocl/core.hpp>
 #include "backends/ocl/goclcore.hpp"
+#include <opencv2/gapi/util/throw.hpp>
```

Comment:
Please order the includes properly (public headers mixed with the internal ones)

Reply:
Done